.github/workflows/drizzle_ci.yml

```diff
@@ -82,3 +82,27 @@ jobs:
     - name: Run flake8
       run: |
         flake8 drizzle --count --show-source --statistics
+
+  code_coverage:
+    name: Code Coverage Report
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install -e .[test]
+        python -m pip install pytest-cov
+    - name: Test with pytest and code coverage
+      run: |
+        pytest --cov-report=xml --cov=. --cov-config=setup.cfg
```

Comment:
May want to add some configuration in setup.cfg to make sure coverage is covering what we expect.

Reply:
I changed this to be `--cov-config=./drizzle/tests/coveragerc`.

Reply:
Unfortunately that `coveragerc` file isn't doing anything because it looks like Bernie just copied the template there without running `cookiecutter` on it.  I.e. the `{packagename}` stuff is making it not do anything.

Reply:
You can see it is not having an effect by clicking through to the Codecov report below and looking at covered files:

https://codecov.io/gh/spacetelescope/drizzle/pull/58/tree?path=drizzle